### DISCUSSION TITLE
erro ao baixar arquivo

### DISCUSSION THREAD

**Discussion by adautoar** [aclap-dev/video-downloadhelper#512] [Help and Q&A]
### Your question

quando tento baixar o arquivo aparece essa mensagem

adly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error!

SideDownload error: : [AVFormatContext @ 0000017a186786c0] Unable to choose an output format for 'C:\Users\rodri\dwhelper\Nutror-app nutror.mp2t'; use a standard extension for the filename or specify the format manually.
[out#0 @ 0000017a19fc0100] Error initializing the muxer for C:\Users\rodri\dwhelper\Nutror-app nutror.mp2t: Invalid argument
Error opening output file C:\Users\rodri\dwhelper\Nutror-app nutror.mp2t.
Error opening output files: Invalid argument
como resolver?

### Addon & CoApp version and details

adly we failed at downloading that specific media. We try to support as many websites as possible, so it would help a lot if you could report that error!

SideDownload error: : [AVFormatContext @ 0000017a186786c0] Unable to choose an output format for 'C:\Users\rodri\dwhelper\Nutror-app nutror.mp2t'; use a standard extension for the filename or specify the format manually.
[out#0 @ 0000017a19fc0100] Error initializing the muxer for C:\Users\rodri\dwhelper\Nutror-app nutror.mp2t: Invalid argument
Error opening output file C:\Users\rodri\dwhelper\Nutror-app nutror.mp2t.
Error opening output files: Invalid argument

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

There are a couple of problems with your problem report.  You did not follow the instructions on the page where you created your problem report.  You need to follow the instructions for supplying your versions of the add-on & the CoApp.  You also need to supply the URL of the web page on which your problematic video is hosted.  Click the 3 dots above & to the right of your post.  Then correct your post by supplying that information.

Also, you need to be running the latest release of VDH, which you can find by clicking the Wiki link at the top of this page.  The Wiki is a sort of user manual.  You should get in the habit of referring to that frequently.  In the Wiki, look for the link to VDH beta versions & the link for CoApp installation.  You need to install the latest versions of both.
